src/js/index.js

```diff
@@ -47,22 +64,51 @@ iconcreate.addEventListener('click', () => {
 })
 
 iconprofile.addEventListener('click', () => {
-  pagehome.classList.add('hidden')
-  pagebookmark.classList.add('hidden')
-  pagecreate.classList.add('hidden')
-  pageprofile.classList.remove('hidden')
+  pagehome.hidden = true
+  pagebookmark.hidden = true
+  pagecreate.hidden = true
+  pageprofile.hidden = false
 
   iconhome.classList.remove('active-icon')
   iconbookmark.classList.remove('active-icon')
   iconcreate.classList.remove('active-icon')
   iconprofile.classList.add('active-icon')
-}) */
+})
 
-iconhome.addEventListener('click', () => {
-  pagehome.hidden = false
-  pagebookmark.hidden = true
-  pagecreate.hidden = true // Boolean: true or false
-  pageprofile.hidden = true
+bookmarkIcons.forEach(icon => {
+  icon.addEventListener('click', () => {
+    icon.classList.toggle('activebookmark')
+  })
+})
+
+cards.forEach(card => {
+  const buttonShowAnswer = card.querySelector('[data-js="buttonShowAnswer"]')
+  const answer = card.querySelector('[data-js="answer"]')
+
+  buttonShowAnswer.addEventListener('click', () => {
+    answer.classList.toggle('hidden')
+  })
+})
+
+submitButton.addEventListener('submit')(event => {
+  event.preventDefault()
+  console.log(event)
+  form.reset()
+  inputQuestion.focus()
+})
+
+submitButton.addEventListener('click', () => {
+  submitText.hidden = false
```

Comment:
Nice idea! 
Please add that to the `form.addEventListener('submit', event => { ...` callback.